src/Covid19Dashboard/ReduxCovid19Dashboard.js

```diff
@@ -10,6 +10,10 @@ async function handleChartData(propName, data) {
   case 'seirObserved':
   case 'seirSimulated':
     return formatSeirData(data);
+  case 'top10':
+    return formatTop10Data(data);
+  case 'idphDaily':
+    return formatTop10Data(data); // change the name
```

Comment:
let's change the name then? :p

Reply:
Changed to `readSingleColumnTSV` and `readMultiColumnTSV`.